coordinator/src/services/translation/request/agent-unattended.ts

```diff
@@ -41,7 +41,7 @@ function createAgentPersonPersonForUnattendedAccess(): hl7V3.AgentPersonPerson {
   return agentPerson
 }
 
-async function createRepresentedOrganization(
+export async function createRepresentedOrganization(
```

Comment:
this isn't necessarily code that is agent-unattended, can we move this to a common file?

Reply:
a lot of the code is duplicate, we could potentially have a `createAuthor` method that can optionally have a professional code? then all of the woolly outside waffle gets reduced into just the one function

Reply:
however, in the longer term when we have more than just the professional code passed in to the message, i expect we will be receiving a practitionerRole resource here instead. either way can reduce duplication